.gitignore

```diff
@@ -42,3 +42,7 @@ profile.out
 
 # mobile related files
 cmd/aries-agent-mobile/build/
+
+# node.js modules
+node_modules/
+package-lock.json
```

Comment:
I think the package-lock.json should not be ignored for folders that become npm modules.
Particularly https://github.com/hyperledger/aries-framework-go/tree/main/cmd/aries-js-worker

I've been somewhat on the fence about test directories containing package-lock.json.

Reply:
Thank you for your reply and remind. ^^
I did not notice that , thank you.